app/models/coin.rb

```diff
@@ -52,6 +52,72 @@ def self.symbols
     pluck(:symbol).uniq.compact.sort.reject { |symbol| /[[:lower:]]/.match(symbol) }
   end
 
+  def most_common_news_category
+    NewsCategory
+      .joins(news_items: :news_coin_mentions)
+      .where(news_coin_mentions: { coin_id: self })
+      .group(:id)
+      .order('COUNT(DISTINCT news_items.id) DESC')
+      .limit(1)
+      .first()
+  end
+
+  def most_common_feed_source
+    FeedSource
+      .joins(news_items: :news_coin_mentions)
+      .where(news_coin_mentions: { coin_id: self })
+      .group(:id)
+      .order('COUNT(DISTINCT news_items.id) DESC')
+      .limit(1)
+      .first()
```

Comment:
Let's stick with Ruby conventions of no parens for function calls without parameters.